docs/expo_development.md

```diff
@@ -0,0 +1,87 @@
+## Overview
+
+The RUM React Native SDK supports Expo and Expo Go. The minimum supported version is [**@datadog/dd-sdk-reactnative:1.0.0-rc9**][1]. 
+
+Datadog recommends using **Expo SDK 45** as a minimum version; previous versions may require manual steps.
+
+## Setup
+
+No configuration plugins are needed. To get started, see [React Native Monitoring][2].
+
+## Expo Go
+
+If you are using Expo Go, switch to development builds (recommended), or keep using Expo Go without Datadog while having it run on your standalone application (not recommended).
+
+### Switch from Expo Go to development builds
+
+Your application's [development builds][3] are debug builds that contain the `expo-dev-client` package.
+
+1. Enable the [custom native code to run][4] with `expo run:android` and `expo run:ios`.
+2. Build your application with the [`expo-dev-client` package][5] to execute your added native code in dev mode.
+3. To start using your development application, run `expo install expo-dev-client` and `expo start --dev-client`.
```

Comment:
```suggestion
2. To start using your development application, run `expo install expo-dev-client` and `expo start --dev-client`. This will install and start the [`expo-dev-client` package][5] to execute the added native code in dev mode.
```

Step 2 was the description of step 3.